### ISSUE TITLE
frank publish puts frankexp-projectname directory on server

### ISSUE THREAD

**Issue by Peeja** [trvsdnn/frank#37]
I'm running on on nearlyfreespeech.net, which serves content out of `/home/public`, so that's what I set my `Frank.publish.path` to.  When I run `frank publish`, Frank puts my site in a subdirectory within `/home/public`.  I can't configure what directory I serve my files from.  Is there a way to make Frank leave the files directly in my `Frank.publish.path`?

**Comment by trvsdnn** [comment]
It's a result of the way the ssh library handles scp. The only way to really handle it (that i know of) is to have frank delete the folder first. I opted out of that because it seemed dangerous/destructive.

I really just need to re-vist publish for the next release. I would recommend just writing a small shell or ruby script to do frank export and then rsync the exported folder.

I'll close this whenever I have a solid solution.

**Comment by Peeja** [comment]
Could it scp each file in the directory instead of the directory itself?

**Comment by trvsdnn** [comment]
Yeah, but you still run into the problem of stale files. You'd need to delete them on the server before trying to copy the new ones up. I'm open to thoughts/ideas on it though.

**Comment by je** [comment]
i know nothing about ruby, but i removed "frankexp-" from the paths in:

\lib\frank\publish.rb
\spec\publish_spec.rb

and it publishes to the projectname folder now, which is what i wanted; maybe removing #projectname from the same would get you to root.
